samples/Exporters/Web/appsettings.json

```diff
@@ -7,7 +7,9 @@
     }
   },
   "AllowedHosts": "*",
-  "Zipkin": {
-    "Endpoint": "http://localhost:9411/api/v2/spans"
+  "Jaeger": {
+    "ServiceName": "jaeger-test",
+    "Host": "localhost",

```

Comment:
Side note: it feels strange that we need to set Jaeger Host and Port to what it should be default values. Anyway, to be addressed in a separate PR.